### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by raghavan2004** [scraly/understanding-docker-in-a-visual-way#1]
I downloaded the docker material and went through it.Its was too good for a beginner and nice. Thanks a lot for the material like this.

I am working as a wordpress designer with Divi. I need guidance from you on how to do it.

I want to create a docker container and install wordpress and mysql but how do i automate and replace the username, password and the mysql credentials in the wp-config.php fille.

Regards,
Vijey

**Comment by scraly** [comment]
Hi @raghavan2004 
GH didn't notidy me about this message. have you found a solution?

**Comment by raghavan2004** [comment]
No, I didn't find any, So I tried with the shell script but I am unable to configure the mysql database creation, trying with wp-cli too no luck.. I will share my code, will you be able to tell me where the mistake is.
Actually I followed the below link to install mysql. Hope this helps. I will share you the code tonight, actually I dont have the code now. I have it in my PC at home..
https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-on-ubuntu-22-04
